docs/services/aws-services-guide.md

```diff
@@ -506,212 +239,101 @@ that we configured earlier.
 * You can verify creation with the following command and output:
 
 ```bash
-$ kubectl get rdsinstanceclass -n aws-infra-dev
+$ kubectl get rdsinstanceclasses.database.aws.crossplane.io
 NAME                 PROVIDER-REF    RECLAIM-POLICY   AGE
-aws-mysql-standard   aws-provider    Delete           11s
+standard-mysql       aws-provider    Delete           11s
 ```
 
 You are free to create more AWS `RDSInstanceClass` instances to define more
 potential configurations. For instance, you may create `large-aws-rds` with
 field `size: 100`.
 
-#### Application Namespaces
-
-Earlier, we created a namespace to group our AWS infrastructure resources.
-Because our application resources may be satisfied by services from any cloud
-provider, we want to separate them into their own namespace. For this demo, we
-will create a namespace called `app-project1-dev`, which we will use to group
-our Wordpress resources.
-
-* Define a `Namespace` in `app-project1-dev-namespace.yaml` and create it:
-
-  ```yaml
-  cat > app-project1-dev-namespace.yaml <<EOF
-  ---
-  apiVersion: v1
-  kind: Namespace
-  metadata:
-    name: app-project1-dev
-  EOF
-
-  kubectl apply -f app-project1-dev-namespace.yaml
-  ```
-
-* You should see the following output:
-
-  > namespace/app-project1-dev created
-
-#### Portable Resource Classes
-
-Portable resource classes are used to define a class of service in a single
-namespace for an abstract service type. We want to define our AWS
-`RDSInstanceClass` as the standard MySQL class of service in the namespace that
-our Wordpress resources will live in.
-
-* Define a `MySQLInstanceClass` in `mysql-standard.yaml` for namespace
-  `app-project1-dev` and create it:
-
-  ```yaml
-  cat > mysql-standard.yaml <<EOF
-  ---
-  apiVersion: database.crossplane.io/v1alpha1
-  kind: MySQLInstanceClass
-  metadata:
-    name: mysql-standard
-    namespace: app-project1-dev
-  classRef:
-    kind: RDSInstanceClass
-    apiVersion: database.aws.crossplane.io/v1alpha2
-    name: aws-mysql-standard
-    namespace: aws-infra-dev
-  EOF
-
-  kubectl apply -f mysql-standard.yaml
-  ```
-
-* You should see the following output:
-
-  > mysqlinstanceclass.database.crossplane.io/mysql-standard created
-
-* You can verify creation with the following command and output:
-
-  ```bash
-  $ kubectl get mysqlinstanceclasses -n app-project1-dev
-  NAME             AGE
-  mysql-standard   27s
-  ```
+#### Resource Claims
 
-Once again, you are free to create more `MySQLInstanceClass` instances in this
-namespace to define more classes of service. For instance, if you created
-`mysql-aws-large` above, you may want to create a `MySQLInstanceClass` named
-`mysql-large` that references it. You may also choose to create MySQL resource
-classes for other non-AWS providers, and reference them for a class of service
-in the `app-project1-dev` namespace.
+Resource claims are used to create external resources by being scheduled to a
```

Comment:
@prasek per our conversation on Slack, I have added slight differences to how each services guide creates a claim that matches to the desired class. I think this helps demonstrate the power of scheduling / defaulting. See the `Resource Claim` sections in the other two guides for context.

Reply:
@hasheddan thanks, that definitely helps. Suggest rewording this section for readability by a new user.

e.g.  Resource claims are used for dynamic provisioning of a managed resource (like a MySQL instance) by matching the claim to a resource class. This can be done in several ways: (a) rely on the default class marked  `resourceclass.crossplane.io/is-default-class: "true"`, (b) use a `claim.spec.classRef` to a specific class, or (c) match on class labels using a `claim.spec.classSelector`.

Note: claims may also be used in [static provisioning](link to quickstart guide) with a reference to an existing managed resource.